frontend/src/Routes/testK8s/DetermineNamespace.tsx

```diff
@@ -18,7 +18,7 @@ const DetermineNamespace: React.FC<DetermineNamespaceProps> = ({ namespace, setN
         model: ProjectModel,
       })
         .then((items) => {
-          const ns = items[0]?.metadata.name;
+          const ns = items[0]?.metadata?.namespace;
```

Comment:
If you'd look at what comes from the API it has the `name` and not `namespace`. Unless it's a bug in API I wouldn't change it (with this change the test k8s page doesn't work at all)

```suggestion
          const ns = items[0]?.metadata.name;
```

Reply:
reverted